app/Components/SudokuBoard/SudokuBoard.tsx

```diff
@@ -97,6 +123,69 @@
     });
   };
 
+  /**
+   * Provides a hint for the current sudoku puzzle state by determining the next possible move
+   * based on the specified strategy order. The hint is generated using the current puzzle state,
+   * solution, and an updated strategy array which prioritizes the "AMEND_NOTES" strategy.
+   *
+   * Updates the hint statistics, including total hints used and hints used per strategy.
+   * Clears the currently selected cells on the board to prepare for hint visualization.
+   *
+   * The hint information is stored in the component's state, including the hint stage and
+   * maximum stages for hint visualization.
+   */
+  const getHint = () => {
+    const updatedArray: SudokuStrategyArray = JSON.parse(
+      JSON.stringify(strategyHintOrderSetting)
+    );
+
+    // prioritize "AMEND_NOTES" and "SIMPLIFY_NOTES"
+    updatedArray.unshift("SIMPLIFY_NOTES");
+    updatedArray.unshift("AMEND_NOTES");
+
+    const returnedHint = getSudokuHint(
+      sudokuBoard.puzzle,
+      sudokuBoard.puzzleSolution,
+      updatedArray
+    );
+
```

Comment:
We shouldn't have any runtime errors, since we know what getSudokuHint returns, and if that changes our test cases will catch that. I don't believe we need this extra validation.